vlib/time/README.md

```diff
@@ -1,17 +1,117 @@
+# time
+
+A time library
+
+## Features
+
+TODO
+
 ## Description:
 
 `time` provides utilities for working with time and dates:
+
 - parsing of time values expressed in one of the commonly used standard time/date formats
 - formatting of time values
 - arithmetic over times/durations
 - converting between local time and UTC (timezone support)
 - stop watches for accurately measuring time durations
 - sleeping for a period of time
 
-## Examples:
+**Example:**
+time can be comparable
+
+see: [V Playground](https://play.vlang.io/p/c121a6dda7)
 
 ```v
 import time
 
 println(time.now())
 ```
+
+### format
+
+**Example:**
+
+see: [V Playground](https://play.vlang.io/p/133d1a0ce5)
+
+```v
+import time
+
+const (
+	time_to_test = time.Time{
+		year: 1980
+		month: 7
+		day: 11
+		hour: 21
+		minute: 23
+		second: 42
+		microsecond: 123456
+		unix: 332198622
+	}
+)
+
+println(time_to_test.format())
+
+assert '1980-07-11 21:23' == time_to_test.format()
+assert '1980-07-11 21:23:42'== time_to_test.format_ss()
```

Comment:
you can run `VAUTOFIX=1 v check-md vlib/time/README.md` to format all of the examples